packages/datatrak-web/index.html

```diff
@@ -4,9 +4,13 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no maximum-scale=1"  />
     <meta name="theme-color" content="#000000" />
+    <!-- Open Graph for Facebook link preview -->
+    <meta property="og:image" content="https://datatrak.tupaia.org/social-preview.png" />
```

Comment:
```suggestion
    <meta property="og:image" content="https://datatrak.tupaia.org/social-preview.png" />
    <meta property="og:image:type" content="image/png" />
```

Reply:
Or these ones but they are a good idea so I will add them all in